source/code-of-conduct.html

```diff
@@ -25,35 +30,44 @@
+                or following; harassing photography or recording, unwanted photography or recording,
+                sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.
             </p>
             <p>
-                Our expectation is that we will treat each other, and others our community interacts with, with consideration, empathy, and respect. Our speech and behavior will be guided by a desire to create and maintain an inclusive community purposed towards sharing knowledge and encouraging enthusiasm for solving problems with PHP.
+                Our expectation is that we will treat each other with consideration, empathy, and respect.
+                Our speech and behavior will be guided by a desire to create and maintain an inclusive
+                community purposed towards sharing knowledge and encouraging enthusiasm for solving problems with PHP.
             </p>
 
             <h2>Consequences of Unacceptable Behavior</h2>
             <p>
-                Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated.
+                Unacceptable behavior from any attendee, including sponsors and those with decision-making authority, will not be tolerated.
             </p>
             <p>
-                Anyone asked to stop unacceptable behavior is expected to comply immediately. If a community member engages in unacceptable behavior, the conference organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
+                Anyone asked to stop unacceptable behavior is expected to comply immediately. If an attendee engages in
+                unacceptable behavior, the conference organizers may take any action they deem appropriate, up to and including
+                a temporary ban or permanent expulsion from the conference without warning (and without refund).
             </p>
 
             <h2>Complaints</h2>
             <p>
-                It is every community member's responsibility to notice, and attempt to intervene or notify others, if unacceptable behaviors are occurring at a meeting or event.
-            </p>
-            <p>
-                If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. You can contact the organizers at admins@seaphp.com, send a direct message to @seaphp on twitter, or speak to an organizer in person at an event.
+                If you are subject to or witness unacceptable behavior, or have any other concerns, please
+                notify an organizer as soon as possible. You can contact the organizers at admins@seaphp.com,
+                send a direct message to @seaphp on twitter, or speak to an organizer in person.
```

Comment:
This contact info probably needs to be updated. Both the email and the twitter handle?